app/serializers/comment_serializer.rb

```diff
@@ -1,5 +1,3 @@
-class CommentSerializer < ActiveModel::Serializer
-  attributes :id, :user, :content, :votes_count, :created_at, :updated_at
+class CommentSerializer < MainSerializer
```

Comment:
Given that this is empty, we can delete it and have `comments` use `MainSerializer` directly.